Source/Mosa.Utility.Launcher/AppLocationsSettings.cs

```diff
@@ -40,14 +40,22 @@ private static string FindQemu()
 		return TryFind(
 			new string[] { /*"qemu-system-x86_64", "qemu-system-x86_64.exe", */"qemu-system-i386.exe", "qemu-system-i386" },
 			new string[] {
-				@"%MOSA%\Tools\QEMU",
```

Comment:
Undo this. We need to test against a stable/working release of QEMU. Thanks.

Reply:
What do you mean by that? I simply changed the path from just QEMU to QEMU\Windows, I didn't remove any.